docs/api/safe-storage.md

```diff
@@ -4,7 +4,20 @@
 
 Process: [Main](../glossary.md#main-process)
 
-This module protects data stored on disk from being accessed by other applications or users with full disk access.
+This module adds extra protection to data being stored on disk by using OS provided cryptography systems. Current
+security semantics for each platform are outlined below.
+
+* **macOS**: Encryption keys are stored for your app in Keychain Access in a way that without user override prevents
+other applications from loading them. Thus protecting content from other users and other apps running in the same userspace.
```

Comment:
dumb question, but is there any documentation we can point to (or added explanation) about what "apps running in the same userspace" means? Does that mean that an application running as Root will bypass this restriction?

Reply:
> Does that mean that an application running as Root will bypass this restriction?

No, but on macOS root can be used to sign in to other users so 🤷